ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml

```diff
@@ -4,6 +4,7 @@ chain:
   - ref: cucushift-helper-service-build-info
   - chain: cucushift-installer-check-cluster-health
   - ref: idp-htpasswd
+  - ref: fips-check-fips-or-die
```

Comment:
@jianzhangbjz I have several questions. Could you please help to clarify? Thanks.
1. Does the script works for all platforms? amd64, arm,  heterogeneous, ppc64? 
2. Does the script works for disconnected/private env?
3. If added the script here, do we you want to remove related test cases from openshift-test-private repo? Or you want to keep both

Reply:
> Does the script works for all platforms? amd64, arm, heterogeneous, ppc64?

No, only for amd64.

> Does the script works for disconnected/private env?

Yes.

> If added the script here, do we you want to remove related test cases from openshift-test-private repo? Or you want to keep both

Yes, I will remove related test cases of openshift-test-private repo once ready.

Reply:
@jianzhangbjz Double confirm one question: I noticed the chain openshift-e2e-test-qe will be called in a lot of openshift-openshift-tests-private-release-4.x__xdevpreview.yaml, such as below. Will it fail or blocking test when platform is not amd64? Thanks.
`https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-devpreview.yaml#L108`

Reply:
> > Does the script works for all platforms? amd64, arm, heterogeneous, ppc64?
> 
> No, only for amd64.
>Double confirm one question: I noticed the chain openshift-e2e-test-qe will be called in a lot of openshift-openshift-tests-private-release-4.x__xdevpreview.yaml, such as below. Will it fail or blocking test when platform is not amd64? Thanks.
`https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-devpreview.yaml#L108`
> > Does the script works for disconnected/private env?
> 
> Yes.
> 
> > If added the script here, do we you want to remove related test cases from openshift-test-private repo? Or you want to keep both
> 
> Yes, I will remove related test cases of openshift-test-private repo once ready.

Reply:
> Will it fail or blocking test when platform is not amd64? Thanks.

Hi @xiaojiey, no won't block it. It will skip them if they're not amd64 platform.